examples/common/avb.c

```diff
@@ -80,6 +80,8 @@ int pci_connect(device_t * igb_dev)
 		err = igb_attach_tx(igb_dev);
 		if (err) {
 			printf("attach_tx failed! (%s)\n", strerror(errno));
+      // If attach passed but attach_tx failed, need to detach
+      if (!err) igb_detach(igb_dev);
```

Comment:
Good call, pushed from xcode where it looked fine--will be sure to keep an eye on that in the future!